recipes/pyScienceMode/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pysciencemode" %}
+{% set version = "1.1.4" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pysciencemode-{{ version }}.tar.gz
+  sha256: 91952b59df0ab6b7472c302fd0b5777e271958e57c48d00d063ca1769faa70e6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+
+requirements:
+  host:
+    - python >={{ python_min }}
```

Comment:
```suggestion
    - python {{ python_min }}
```

Reply:
Sorry I missed that earlier. :grimacing:

Reply:
No worries 😃